.github/workflows/update-license-year.yml

```diff
@@ -0,0 +1,16 @@
+name: Update copyright year in license file
+
+on:
+  schedule:
+    - cron: "0 0 2 1 *"
```

Comment:
Nice. Presume this will work. Did you test it?

Reply:
Er, no; I'd have to use a different crontab entry, and we've already changed the license date. :grin: I basically copied the suggested configuration verbatim, and just changed the date and time to midnight, January 2 to avoid any time zone difficulties.

Reply:
Okay. I guess we'll find out in January. :)